### PR TITLE
LPD-30599 [Marketplace] Adjusted behavior of Download Agreement Marketplace

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

13 files changed, 169 insertions(+), 266 deletions(-)

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/AppToolBar/AppToolbar.scss [hunk 1]
```diff
@@ -82,6 +83,7 @@
 		}
 	}
 
+
```
Comment (joanagrego) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/AppToolBar/AppToolbar.scss:86]: Blank line

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/AppToolBar/AppToolbar.scss [hunk 2]
```diff
@@ -3,6 +3,7 @@
 		align-items: center;
 		display: flex;
 		justify-content: center;
+		margin-bottom: 16px;
 		width: 100%;
```
Comment (joanagrego) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/AppToolBar/AppToolbar.scss:7]: Can we use bootstrap here?

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/components/LicenseAgreement/LicenseAgreement.tsx [hunk 3]
```diff
@@ -36,21 +40,24 @@ const LicenseAgreement = () => {
...
 					>
 						{i18n.translate('download')}
 						<ClayIcon className="ml-2" symbol="download" />
 					</ClayButton>
 				</div>
 
-				<div className="p-4 text-agreement">
-					<strong>LIFERAY MARKETPLACE DEVELOPER AGREEMENT</strong>
+				<div className="p-5 text-agreement">
+					<strong className='text-agreement-text-primary'>LIFERAY MARKETPLACE DEVELOPER AGREEMENT</strong>
```
Comment (joanagrego) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/AdministratorDashboard/components/LicenseAgreement/LicenseAgreement.tsx:58]:
```suggestion
					<strong className="text-agreement-text-primary">LIFERAY MARKETPLACE DEVELOPER AGREEMENT</strong>
```

**Review by joanagrego — CHANGES_REQUESTED** [review]

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**